### PR TITLE
extract alicloud plugin to its own repo

### PR DESCRIPTION
**Pull request by SwampDragons**
extracts the alicloud builder; new plugin can be found at 

https://github.com/hashicorp/packer-plugin-alicloud

### CODE REVIEW

**Review by azr — APPROVED** [review]
👍🏼

File: website/data/docs-nav-data.json [hunk 1]
```diff
@@ -679,10 +679,6 @@
         "title": "Overview",
         "path": "builders"
       },
-      {
-        "title": "Alicloud ECS",
-        "path": "builders/alicloud-ecs"
-      },
       {
```
Comment (sylviamoss) [inline website/data/docs-nav-data.json:682]: This file still contains the post-processor. 
There's more than one link in the sidebar https://packer-5crayggmd-hashicorp.vercel.app/docs/post-processors/alicloud-import

**Review by sylviamoss — CHANGES_REQUESTED** [review]
LGTM! 👍🏼  You forgot to remove the old post-processor link from the navbar. 

On the plugin side, I think you need to add the `nav_title`  option to keep that title how currently it is for both builder and post-processor.

File: website/data/docs-remote-plugins.json [hunk 2]
```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "title": "Alicloud",
+    "path": "alicloud",
+    "repo": "hashicorp/packer-plugin-alicloud",
+    "version": "latest"
+  },
```
Comment (nywilken) [inline website/data/docs-remote-plugins.json:7]: With the latest master we can now mark plugins as community

```suggestion
    "pluginTier": "community",
    "version": "latest"
  },
```

**Review by nywilken — APPROVED** [review]
Looks good. Just one suggestion to mark the Alicloud docs as community.

**Comment by SwampDragons** [comment]
Done. Thanks Wilken!

**Pull request merged**